--- setup.py    Mon Jul 29 17:04:25 2013
+++ setup_ch.py Wed Aug 07 10:16:06 2013
@@ -140,7 +140,8 @@
                     # process manually python file in include_dirs (test data)
                     from subprocess import check_call
                     print('running 2to3 on', dest) # parens are NOT optional here for py3k compat
-                    check_call(['2to3', '-wn', dest])
+                    check_call([sys.executable, os.path.join(os.path.dirname(sys.executable), 'Tools', 'Scripts', '2to3.py'), '-wn', dest])
+                    #check_call(['2to3', '-wn', dest])

 def install(**kwargs):
     """setup entry point"""
