--- setup.py	Wed Apr 17 11:38:31 2013
+++ setup_ch.py	Tue Jul 16 23:35:10 2013
@@ -55,7 +55,8 @@
 scripts = getattr(__pkginfo__, 'scripts', [])
 data_files = getattr(__pkginfo__, 'data_files', None)
 subpackage_of = getattr(__pkginfo__, 'subpackage_of', None)
-include_dirs = getattr(__pkginfo__, 'include_dirs', [])
+#include_dirs = getattr(__pkginfo__, 'include_dirs', [])
+include_dirs = ['brain']
 ext_modules = getattr(__pkginfo__, 'ext_modules', None)
 install_requires = getattr(__pkginfo__, 'install_requires', None)
 dependency_links = getattr(__pkginfo__, 'dependency_links', [])
@@ -140,7 +141,7 @@
                     # process manually python file in include_dirs (test data)
                     from subprocess import check_call
                     print('running 2to3 on', dest) # parens are NOT optional here for py3k compat
-                    check_call(['2to3', '-wn', dest])
+                    #check_call(['2to3', '-wn', dest])
 
 def install(**kwargs):
     """setup entry point"""
